bin/setup/lang/gwf_1.htm

```diff
@@ -76,147 +76,147 @@ <h1 style="margin:0;">
 </td>
 </tr>
 <tr align=left>
-<td bgcolor="#FFFFFF">
+<td background-color="#FFFFFF">
 <input type=radio name=proposed_body_prop
-value="bgcolor=#FFFFFF text=#333333 link=#006699 vlink=#999900">
+value="background-color=#FFFFFF text=#333333 link=#006699 vlink=#999900">
 <font color="#333333">[text|link|vlink]0</font>
 <font color="#006699">[text|link|vlink]1</font>
 <font color="#999900">[text|link|vlink]2</font>
 </td>
-<td bgcolor="#FFFFEE">
+<td background-color="#FFFFEE">
```

Comment:
do this work?
background-color is a CSS property shouldn't it be set with `style="background-color: #FFFFEE"`?